inference-engine/tests/unit/inference_engine/ie_blob_proxy.cpp

```diff
@@ -237,7 +226,7 @@ TEST_F(BlobProxyTests, throwsIfSmallProxyObjectSize) {
         TBlobProxy<uint8_t> proxy(Precision::U8, C, b, 0, { b->byteSize() + 1 });
         FAIL() << "Should have failed by now: proxy size is larger than blob size";
     }
-    catch (InferenceEngine::details::InferenceEngineException ex) {};
+    catch (InferenceEngine::details::InferenceEngineException ex) {}
```

Comment:
**Also not a blocker for the merge since this PR mostly for moving the tests:** but `ASSERT_THROW` instead of `try` - `catch` will look more elegant.

Reply:
Updated